docs/reference/ingest/apis/geoip-stats-api.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[geoip-stats-api]]
+=== GeoIP stats API
+++++
+<titleabbrev>GeoIP stats</titleabbrev>
+++++
+
+Gets download statistics for GeoIP2 databases used with the
+<<geoip-processor,`geoip` processor>>.
+
+[source,console]
+----
+GET _ingest/geoip/stats
+----
+
+[[geoip-stats-api-request]]
+==== {api-request-title}
+
+`GET _ingest/geoip/stats`
+
+[[geoip-stats-api-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the `monitor` or
+`manage` <<privileges-list-cluster,cluster privilege>> to use this API.
+
+* If <<ingest-geoip-downloader-enabled,`ingest.geoip.downloader.enabled`>> is
+disabled, this API returns zero values and an empty `nodes` object.
+
+[role="child_attributes"]
+[[geoip-stats-api-response-body]]
+==== {api-response-body-title}
+
+`stats`::
+(object)
+Download statistics for all GeoIP2 databases.
++
+.Properties of `stats`
+[%collapsible%open]
+====
+`successful_downloads`::
+(integer)
+Total number of successful database downloads.
+
+`failed_downloads`::
+(integer)
+Total number of failed database downloads.
+
+`total_download_time`::
+(integer)
+Total milliseconds spent downloading databases.
```

Comment:
> Total milliseconds spent downloading databases.

@probakowski  Can you confirm this is milliseconds? It looks like it in the source, but I had a hard time teasing it out. Thanks!

Reply:
Yes, this is in milliseconds.